docs/schema.md

```diff
@@ -0,0 +1,186 @@
+# Schema Registry
+
+KoP has implemented the Schema Registry to support Kafka clients with [Confluent's Schema serializers and deserializers](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#supported-formats).
+
+To enable the schema registry, you should add the following configuration:
+
+```properties
+kopSchemaRegistryEnable=true
+```
+
+The Schema Registry listens on the 8001 port by default, which means you should add the following property to create your Kafka producer or consumer.
+
+```properties
+schema.registry.url=http://<broker-ip>:8001
+```
+
+> **Note**
+>
+> See [here](./configuration.md) for other configurations, like how to change the port it listens on.
+
+To manage the schemas, see the [Confluent's Schema REST API](https://docs.confluent.io/platform/current/schema-registry/develop/api.html#compatibility).
+
+## Example: Use KoP Schema Registry with Confluent's Avro serializer
+
+This section adds an example to show how to use Confluent's Avro serializer on KoP.
+
+### Enable Schema Registry in KoP standalone
+
+Follow the **Set configuration for KoP** section in [here](./kop.md), start the KoP standalone with the following configurations:
+
+```properties
+messagingProtocols=kafka
+kafkaListeners=PLAINTEXT://0.0.0.0:9092
+brokerEntryMetadataInterceptors=org.apache.pulsar.common.intercept.AppendIndexMetadataInterceptor
+brokerDeleteInactiveTopicsEnabled=false
+allowAutoTopicCreationType=partitioned
+
+# Enable the transaction to be compatible with Kafka clients 3.2.0 or later
+# See https://kafka.apache.org/documentation/#upgrade_320_notable
+kafkaTransactionCoordinatorEnabled=true
+brokerDeduplicationEnabled=true
+
+kopSchemaRegistryEnable=true
+```
+
+### Define the Avro schema
+
+
+Create an Avro schema file named `User.asvc` under the `src/main/avro` directory:
```

Comment:
Does this sentence and the sentence below has a sequential order? If yes, i would like to suggesting using an steplist format. Otherwise, ignore my comment.

Reply:
No.